### DISCUSSION TITLE
Unable to create Postgres DB

### DISCUSSION THREAD

**Discussion by adyabajpai21** [outline/outline#5427] [Self Hosting] [answered]
I am trying to install a self-managed outline on an EC2 instance. 
My docker.env file: 

`NODE_ENV=production

SECRET_KEY={Key}

UTILS_SECRET={Key}

DATABASE_URL=postgres://{username}:{password}@{public IP of instance}/outline
DATABASE_URL_TEST=postgres://{username}:{password}@{public IP of instance}/outline-test

REDIS_URL=redis://{public IP of instance}:6379

URL=http://{public IP of instance}:3000
PORT=3000

AWS_ACCESS_KEY_ID={AWS Key}
AWS_SECRET_ACCESS_KEY={Secret Key}
AWS_REGION=ap-south-1
`

My docker-compose.yml file: 
`
version: "3"
services:

  outline:
    image: docker.getoutline.com/outlinewiki/outline:0.61.1
    env_file: ./docker.env
    ports:
      - "3000:3000"
    depends_on:
      - postgres
      - redis
      - storage

  redis:
    image: redis
    env_file: ./docker.env
    ports:
      - "6379:6379"
    volumes:
      - ./redis.conf:/redis.conf
    command: ["redis-server", "/redis.conf"]
    healthcheck:
      test: ["CMD", "redis-cli", "ping"]
      interval: 10s
      timeout: 30s
      retries: 3

  postgres:
    image: postgres
    env_file: ./docker.env
    ports:
      - "5432:5432"
    volumes:
      - database-data:/var/lib/postgresql/data
    healthcheck:
      test: ["CMD", "pg_isready"]
      interval: 30s
      timeout: 20s
      retries: 3
    environment:
      POSTGRES_USER: 'czdev'
      POSTGRES_PASSWORD: 'charge123'
      POSTGRES_DB: 'outline'

  storage:
    image: minio/minio
    env_file: ./docker.env
    ports:
      - "9000:9000"
    entrypoint: sh
    command: -c 'minio server'
    deploy:
      restart_policy:
        condition: on-failure
    volumes:
      - storage-data:/data
    healthcheck:
      test: ["CMD", "curl", "-f", "http://{public IP of instance}/minio/health/live"]
      interval: 30s
      timeout: 20s
      retries: 3

  https-portal:
    image: steveltn/https-portal
    env_file: ./docker.env
    ports:
      - '80:80'
      - '443:443'
    links:
      - outline
      - storage
    restart: always
    volumes:
      - https-portal-data:/var/lib/https-portal
    healthcheck:
      test: ["CMD", "service", "nginx", "status"]
      interval: 30s
      timeout: 20s
      retries: 3
    environment:
      DOMAINS: 'docs.mycompany.com -> http://outline:3000'
      STAGE: 'production'
      WEBSOCKET: 'true'

volumes:
  https-portal-data:
  storage-data:
  database-data:
`

While running this command mentioned in the official documentation: 

`docker-compose run --rm outline yarn db:create --env=production-ssl-disabled`

I get this error: 
`[+] Building 0.0s (0/0)                                                                                                                                                                                            
[+] Creating 2/0
 ✔ Container ubuntu-postgres-1  Running                                                                                                                                                                       0.0s 
 ✔ Container ubuntu-redis-1     Running                                                                                                                                                                       0.0s 
[+] Running 1/0
 ✔ Container ubuntu-storage-1  Started                                                                                                                                                                        0.0s 
[+] Building 0.0s (0/0)                                                                                                                                                                                            
yarn run v1.22.15
error Command "db:create" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.`

**Comment by tommoor** [comment, marked as answer]
The latest release is `v0.69.2` – the image your compose file is pointing to (`0.61.1`) does not have the command `db:create`.
